README.md

```diff
@@ -451,6 +451,10 @@ Default: `thread`
 <dd>A list of `type` objects, (e.g. `type(my_class_instance)` or `MyClass`) that will be serialized using
     `repr()`. Default `[]`
 </dd>
+<dt>scrub_varargs
+</dt>
+<dd>If `True`, variable arguments will be scrubbed. Default `True`.
```

Comment:
This was a question I had when I read the docs you wrote, so really what I'm asking is for you to clarify it in the docs :)